app/models/dor_harvester.rb

```diff
@@ -51,6 +51,11 @@ def indexable_resources
 
   def on_success(resource)
     RecordIndexStatusJob.perform_later(self, resource.bare_druid, ok: true)
+    IndexRelatedContentJob.perform_later(self, resource.bare_druid) if index_related_content?
+  end
+
+  def index_related_content?
+    FeatureFlags.new.for(exhibit.slug).index_related_content?
```

Comment:
You can drop the `new` here if you want: https://github.com/sul-dlss/exhibits/blob/502f4496ffc9c0c2a981912ed71f44763692acef/app/models/feature_flags.rb#L28-L29

Reply:
👍